CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## master / unreleased
 
+* [FEATURE] Adds support to S3 server side encryption using KMS. Deprecated `-<prefix>.s3.sse-encryption`, you should use the following CLI flags that have been added. #3651
```

Comment:
The order of entries should be [CHANGE], [FEATURE], [ENHANCEMENT], [BUGFIX]. Please move this entry after `[CHANGE]` below.